### PR TITLE
Bandwidth addition to interconnect

### PR DESCRIPTION
**Pull request by Madankapoor**
This PR is to allow bandwidth option to be allowed for compute interconnect attachment for dedicated GCP interconnects. It is related to the issue https://github.com/terraform-providers/terraform-provider-google/issues/4109 
# Release Note for Downstream PRs (will be copied)
```releasenote
compute: Adding bandwidth attribute to interconnect attachment.
```

### CODE REVIEW

File: products/compute/terraform.yaml [hunk 1]
```diff
@@ -761,6 +761,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         diff_suppress_func: 'compareSelfLinkOrResourceName'
       interconnect: !ruby/object:Overrides::Terraform::PropertyOverride
         diff_suppress_func: 'compareSelfLinkOrResourceName'
+      bandwidth: !ruby/object:Overrides::Terraform::PropertyOverride
```
[the remark this answers is not in the document]
Comment (Madankapoor) [reply products/compute/terraform.yaml]: Thank you for the Review and feedback.  

Hi I have added default_from_api as true and tested it with a dedicated connection by compiling the terraform provider.

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-2.6.0
+2.6.0-dev
```
Comment (danawillow) [inline .ruby-version]: Is there a reason for this change?

Comment (Madankapoor) [reply .ruby-version]: no it is was issue with my macos with rbenv.  I am using custom docker ruby 2.6.0 with volume binding to work around it now. It should work with ruby 2.6.0.

**Comment by Madankapoor** [comment]
Thanks for the feedback. I have made the default_from_api true in terraform.yaml. I was able to test it with custom terraform module.

**Comment by Madankapoor** [comment]
Hi, I was waiting If this could be merged. I wanted to open another pull request for next-hop Internal Load Balancer route for the beta terraform provider.

**Pull request merged**